projects/dsp-ui/src/lib/viewer/services/value.service.spec.ts

```diff
@@ -262,8 +262,8 @@ describe('ValueService', () => {
 
             const period = calDateJDN.toJDNPeriod();
 
-            expect(period.periodStart).toEqual(1721060);
-            expect(period.periodEnd).toEqual(1721060);
+            expect(period.periodStart).toEqual(1721058);
```

Comment:
@gautschr I had to adapt these expectations.


The Gregorian calendar date of January 1st 1 BCE used to be 1721060 in JDN, now it is 1721058.
Since the date is before 1582, it was calculated like a Julian calendar date, right?

Reply:
Yes!